# HG changeset patch
# User Richard Russon <rich@flatcap.org>
# Date 1458400341 0
#      Sat Mar 19 15:12:21 2016 +0000
# Branch gentoo-1.7
# Node ID 61060006a2ab8031472de6a0f656e9f0dff8f6a1
# Parent  6ee58b6145274110cf5625947f01d5418ae42514
[PATCH] debian: hurd build fix
From c1c0ae0a086893e63e5155fcecd33c9ae547a258 Mon Sep 17 00:00:00 2001
Hurd is missing a definition for PATH_MAX.
---
 mutt.h | 5 +++++
 1 file changed, 5 insertions(+)

diff -r 6ee58b614527 -r 61060006a2ab mutt.h
--- a/mutt.h	Tue Oct 22 11:34:57 2013 -0700
+++ b/mutt.h	Sat Mar 19 15:12:21 2016 +0000
@@ -52,6 +52,11 @@
 #include <limits.h>
 #endif
 
+/* PATH_MAX is undefined on the hurd */
+#ifndef PATH_MAX
+#define PATH_MAX _POSIX_PATH_MAX
+#endif
+
 #include <pwd.h>
 #include <grp.h>
 
